css/_videolayout_default.scss

```diff
@@ -378,6 +378,28 @@
 .remote-video-menu-trigger {
     margin-top: 7px;
 }
+.popover-mousemove-padding-bottom {
+    bottom: -15px;
+    height: 20px;
+    position: absolute;
+    right: 0;
+    width: 100%;
+}
+.popover-mousemove-padding-right {
+    height: 100%;
+    position: absolute;
+    right: -20;
+    top: 0;
+    width: 40px;
+}
+
+.popover-mouse-top-padding {
```

Comment:
the naming is inverted compared to the above (i.e. xxx-padding-right and then xxx-top-padding)